webrtc-stats.html

```diff
@@ -1251,7 +1252,9 @@ <h3>
         </h3>
         <p>
           The <dfn>RTCRemoteInboundRTPStreamStats</dfn> dictionary represents the remote endpoint's
-          measurement metrics for its incoming RTP stream (our outgoing RTP stream).
+          measurement metrics for a particular incoming RTP stream (corresponding to an outgoing
+          RTP stream at the sending endpoint). The timestamp reported in the statistics object is
+          the time at which the corresponding RTCP RR was received.
```

Comment:
Should this say "RTCP RR or SR"? I'm not an expert on RTP, but from what I've gathered, both SR and RRs contain "reception report blocks".

Reply:
The information in this particular remote object is extracted from the RTCP RR. Although it could be a compound RTCP containing both an SR and an RR.